charts/pomerium/README.md

```diff
@@ -82,7 +82,11 @@ In default configuration, this chart will automatically generate TLS certificate
 Upon delete, you will need to manually delete the generated secrets. Example:
 
 ```console
-kubectl delete secret -l app.kubernetes.io/name=pomerium
+kubectl delete secret -l name=pomerium-authenticate-tls
```

Comment:
@tgomas you don't need the `-l name=`, `kubectl delete secret pomerium-authenticate-tls` is enough.

Reply:
done